resources/lib/modules/router.py

```diff
@@ -110,6 +110,8 @@ def dispatch(params):
         trakt.TraktAPI().revoke_auth()
 
     elif action == "getSources":
+        kodi_params = ('{"jsonrpc":"2.0","method":"GUI.ActivateWindow","params":{"window":"busydialog"},"id":"1"}')
```

Comment:
We do already have a method for the busy dialog: https://github.com/nixgates/plugin.video.seren/blob/84c1034cbf926e6354b55110e493374f1a8a8d14/resources/lib/modules/globals.py#L1101

Using that method will save the overhead of making a JSON-RPC call... but I don't know that it's really needed here anyways.

Once again, this isn't related to hoster quality in any way.